HISTORY.md

```diff
@@ -1,13 +1,21 @@
 # Breaking updates and feature summaries across releases
 
 ## Catalyst unreleased (master branch) 
-- **BREAKING:** added a dependency on Nemo.jl for integer matrix linear algebra methods.
-- Added `incidencematgraph`, `linkageclasses`, `deficiency`, `subnetworks` and `linkagedeficiency` API functions.
+- **BREAKING:** added a dependency on Nemo.jl for integer matrix linear algebra
```

Comment:
This is GPL and adds a huge dependency tree IIRC?

Reply:
Should we drop it then? This says LGPL 2.1, but it does seem Nemo has a big mix of stuff so it could be hard to know if one is going into GPL code: https://github.com/Nemocas/Nemo.jl/blob/master/LICENSE.md

Unfortunately AbstractAlgebra doesn’t have exactly the same functionality.

Reply:
In terms of the dependency tree I found it was only twice as slow as AbstractAlgebra at loading (I.e. calling `using`), and that wasn’t anywhere near how long MT takes so not really noticeable.

Reply:
Actually, even AbstractAlgebra’s license statement is confusing, saying it may link to GPL only packages but their source is BSD-2.

Not sure what we should do here. I guess we could remove all the network analysis functionality which relies on integer LA and put it in a separate CatalystNetworkAnalysis package?